includes/Google/u2f-api.js

```diff
@@ -0,0 +1,361 @@
+// Copyright 2014 Google Inc. All rights reserved
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+
+/**
+ * @fileoverview The U2F api.
+ */
+
+'use strict';
+
+/** Namespace for the U2F api.
+ * @type {Object}
+ */
+var u2f = u2f || {};
+
+/**
+ * The U2F extension id
+ * @type {string}
+ * @const
+ */
+u2f.EXTENSION_ID = 'kmendfapggjehodndflmmgagdbamhnfd';
```

Comment:
`u2f.EXTENSION_ID` is pointing *CryptoToken Component* on Google Chome. (not API key)
Dropbox (https://goo.gl/VgCpCB) and Yubico's demo (https://goo.gl/2o7bqN) are using the same js.